roles/softserve/templates/softserve.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=backend server for softserve
+After=network.target
+
+[Service]
+Type=simple
+PIDFile=/run/softserve/softserve.pid
+WorkingDirectory=/srv/softserve
+ExecStart=/softserve/env/bin/gunicorn --pid /run/softserve/softserve.pid -w 4 softserve:app
```

Comment:
The path here is wrong. Should be /srv/softserve, according to tasks/main.yml. And it might also be better to use {{ destination }}